content/projects/tdd/password-checker/part1/_index.md

```diff
@@ -31,28 +31,28 @@ Your directory structure should look like this.
         > support
             -jasmine.json
         - password_is_valid_spec.js
-        - password_is_ok_spec.js
+        - password_strength.js
```

Comment:
Should this not have _spec like the others? Why should the naming convention be different now?